src/me/corriekay/pokegoutil/windows/PokemonTab.java

```diff
@@ -850,10 +862,13 @@ private void constructNewTableModel(PokemonGo go, List<Pokemon> pokes) {
                 caughtCol = new ArrayList<>(),//22
                 favCol = new ArrayList<>();//23
         private final ArrayList<Long> duelAbilityCol = new ArrayList<>();//24
-        private final ArrayList<Long> gymOffenseCol = new ArrayList<>();//25
+        private final ArrayList<Double> gymOffenseCol = new ArrayList<>();//25
```

Comment:
gymOffense potentially has a .5 attached, it was throwing off other values occasionally on me before, took me forever to figure out why some pokemons were just slightly off.